desktop/mac/release-notes/index.md

```diff
@@ -19,14 +19,46 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.2.0
-2021-11-09
+## Docker Desktop 4.3.0
+2021-12-02
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### Upgrades
+
+- [Docker Engine v20.10.11](https://docs.docker.com/engine/release-notes/#201011)
+- [containerd v1.4.12](https://github.com/containerd/containerd/releases/tag/v1.4.12)
+- [Buildx 0.7.1](https://github.com/docker/buildx/releases/tag/v0.7.1)
+- [Compose v2.2.1](https://github.com/docker/compose/releases/tag/v2.2.1)
+- [Go 1.17.3](https://golang.org/doc/go1.17)
+- [Kubernetes 1.22.4](https://github.com/kubernetes/kubernetes/releases/tag/v1.22.4)
+- [hub-tool 0.4.4](https://github.com/docker/hub-tool/releases/tag/v0.4.4)
+
+### Bug fixes and minor changes
+
+- Improved: Docker Desktop on Apple Silicon no longer requires Rosetta 2.
+- Fixed an issue using the "Save As" action in the Volumes UI to save files from a volume. Fixes [docker/for-win#12407](https://github.com/docker/for-win/issues/12407).
+- Fixed applying an update after downloading it was sometimes re-launching the same version instead of the new one.
```

Comment:
```suggestion
- Fixed an issue that sometimes caused Docker Desktop to launch the existing version instead of a new version after downloading the update.
```

Reply:
@trungutt How about "Fixed an issue that sometimes launched the existing version of Docker Desktop even after updating to a newer version."

Reply:
Thanks a lot

Reply:
Should we point out that the fix will only take effect for the next update? 🤔